backend/gvz_api/utils.py

```diff
@@ -0,0 +1,230 @@
+        :type region_name: str
+        :return: JSON search results defined in the GVZ API
+        :rtype: str
+        """
+
+        logger.info("Searching for %s", region_name)
+        response = requests.get("{}/search/{}".format(self.api_url,
+                                                      region_name))
+        regions = json.loads(response.text)
+        return regions
+
+    def get_details(self, region_key):
+        """
+        Get details for a region key (i.e. Gemeindeschlüssel)
+
+        :param region_key: official ID for a region, i.e. Gemeindeschlüssel
+        :type region_key: str
+        :return: dictionary containing longitude, latitude, type, key, name
+        :rtype: dict
+        """
+        logger.info("GVZ API: Details for %s", region_key)
+        response = requests.get("{}/details/{}".format(self.api_url,
+                                                       region_key))
+        region = json.loads(response.text)[0]
+        if ',' in region['name']:
+            region['name'] = region['name'].split(',')[0]
+        return {
+            "key": region['key'],
+            "name": region['name'],
+            "longitude": region['longitude'],
+            "latitude": region['latitude'],
+            "type": region['type']
+        }
+
+    def get_children(self, region_key):
+        """
+        Get keys of children of a region
+
+        :param region_key: official ID for a region, i.e. Gemeindeschlüssel
+        :type region_key: str
+        :return: list of children dictionaries
+        :rtype: list
+        """
+        response = requests.get("{}/searchcounty/{}".format(self.api_url,
+                                                            region_key))
+        content = json.loads(response.text)
+        if content:
+            return content[0]['children']
+        return []
+
+    def translate_type(self, region_type):  # pylint: disable=R0201
+        """
+        Map Integreat CMS region types to Gemeindeverzeichnis region types.
+
+        :param region_type: administrative division type of a region
+        :param type: str
+        :return: administrative division type
+        :rtype: cms.constants.administrative_division
+        """
+        if region_type in ("Stadt", "Kreisfreie Stadt"):
+            return administrative_division.CITY
+        if region_type in ("Landkreis", "Kreis"):
+            return administrative_division.RURAL_DISTRICT
+        return None
+
+    def filter_region_types(self, region_details):
+        """
+        Special filters for certain region types. For example, Kreisfreie
+        Städte exist multiple times on different levels. We want the lower
+        level that contain the coordinates.
+
+        :param region_details: dictionary with region details
+        :type region_details: dict
+        :return: indicates if a region is not interesting
+        :rtype: bool
+        """
+        if self.translate_type(
```

Comment:
```suggestion
    @classmethod
    def filter_region_types(cls, region_details):
        """
        Special filters for certain region types. For example, Kreisfreie
        Städte exist multiple times on different levels. We want the lower
        level that contain the coordinates.

        :param region_details: dictionary with region details
        :type region_details: dict
        :return: indicates if a region is not interesting
        :rtype: bool
        """
        if cls.translate_type(
```
This change is necessary if `translate_type()` is a static method.